app/assets/index.html

```diff
@@ -22,6 +24,9 @@
     </script>
   </head>
   <body>
+    <noscript>
+      <p style="text-align: center;">Please enable JavaScript to use Yugioh Calculator.</p>
```

Comment:
Can you use a CSS class and add more styling here?

A "reload" button might also be nice (i.e. an HTML link pointing to the same page).

I am thinking that a light gray rectangle with rounded corners (like the one used for life points) would be in the top part of the screen instructing to enable JS, and a dark gray rectangle with rounded corners (like the one used for digits) would be in the lower part of the screen and provide the "reload" button.

[General idea.](https://shareitdammit.com/MIoJiIfm)  Feel free to improve or be more creative.